site/testData/mockData.ts

```diff
@@ -420,3 +420,76 @@ export const ptSituationElementWithMultipleConsequences = {
     ReasonType: "PersonnelReason",
     PersonnelReason: "staffInWrongPlace",
 };
+
+export const mockOperators: Operator[] = [
+    {
+        id: 1,
+        nocCode: "1CTL",
+        operatorPublicName: "1st Choice Transport Ltd",
+        vosaPsvLicenseName: "1St Choice Transport Ltd",
+        opId: "135427",
+        pubNmId: "93089",
+        nocCdQual: "",
+        changeDate: "",
+        changeAgent: "",
+        changeComment: "",
+        dateCeased: "",
+        dataOwner: "",
+    },
+    {
+        id: 3,
+        nocCode: "2WTR",
+        operatorPublicName: "2 Way Transport",
+        vosaPsvLicenseName: "2 Way Transport",
+        opId: "135428",
+        pubNmId: "93090",
+        nocCdQual: "",
+        changeDate: "",
+        changeAgent: "",
+        changeComment: "",
+        dateCeased: "",
+        dataOwner: "",
+    },
+    {
+        id: 4,
+        nocCode: "3DCO",
+        operatorPublicName: "3D Coaches",
+        vosaPsvLicenseName: "Peter Kermeen & Elaine Fletcher",
+        opId: "137388",
+        pubNmId: "93092",
+        nocCdQual: "",
+        changeDate: "",
+        changeAgent: "",
+        changeComment: "",
+        dateCeased: "",
+        dataOwner: "",
+    },
+    {
+        id: 5,
+        nocCode: "5STR",
+        operatorPublicName: "Five Star International Travel",
+        vosaPsvLicenseName: "Philip Riley",
+        opId: "137408",
+        pubNmId: "94084",
+        nocCdQual: "",
+        changeDate: "",
+        changeAgent: "",
+        changeComment: "",
+        dateCeased: "",
+        dataOwner: "",
+    },
+    {
+        id: 6,
+        nocCode: "8H",
+        operatorPublicName: "Highland Airways",
+        vosaPsvLicenseName: "Highland Airways",
+        opId: "136620",
+        pubNmId: "94334",
+        nocCdQual: "",
+        changeDate: "",
```

Comment:
Are all of these needed?

Reply:
Its just copy and pasted data that came back from the API.